### DISCUSSION TITLE
请问是否可以提供百度的broker jar包

### DISCUSSION THREAD

**Discussion by jeffkxt** [apache/doris#20933] [A - General / Q&A]
请问一下下载版本1.2.4版本包中没有提供 百度的broker 操作类，导致无法建立百度bos的broker 连接，需要去哪里下载百度bos jar包

**Comment by LemonLiTree** [comment]
you can find it here: https://archive.apache.org/dist/doris/1.2/1.2.4.1-rc01/
broker in apache-doris-dependencies-1.2.4.1-bin-xxx.tar.xz.

**Reply by jeffkxt** [reply]
可否提供一个和百度obs连接的例子

**Comment by jeffkxt** [comment]
Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=137) errCode = 2, detailMessage = Failed to create repository: failed to list remote path: bos://cam-pms/doris_backup/__palo_repository_bos_repo/__repo_info. msg: java.lang.RuntimeException: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.bos.BaiduBosFileSystem not found, cause by: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.bos.BaiduBosFileSystem not found, broker: broker_name

**Reply by jeffkxt** [reply]
我们连接百度obs报错
